internal/testhelpers/helpers_registry.go

```diff
@@ -0,0 +1,30 @@
+package testhelpers
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-cli/v2/core"
+)
+
+func PushRegistryImage(dockerImage, namespaceMetaKey, imageName string) core.BeforeFunc {
+	return func(ctx *core.BeforeFuncCtx) error {
+		namespaceEndpoint := ctx.Meta.Render(fmt.Sprintf("{{ .%s.Endpoint }}", namespaceMetaKey))
+		remote := fmt.Sprintf("%s/%s:latest", namespaceEndpoint, imageName)
```

Comment:
Why is latest hard coded? It could be part of the image name that is passed around